ecs-cli/modules/cli/compose/entity/entity_helper.go

```diff
@@ -120,7 +120,18 @@ func CollectTasksWithStatus(entity ProjectEntity, status string, filterLocal boo
 	result := []*ecs.Task{}
 
 	err := entity.Context().ECSClient.GetTasksPages(request, func(respTasks []*ecs.Task) error {
-		result = append(result, respTasks...)
+		// Filter the results by task.Group
+		if entity.EntityType() == "task" && filterLocal {
```

Comment:
1. Can you use a const for comparison instead of `"task"`? Also, it'd make more sense for `EntityType()` to return an enum instead of a string (https://github.com/aws/amazon-ecs-cli/blob/master/ecs-cli/modules/cli/compose/entity/entity.go#L42)

2. Can you update the comment for `CollectTasksWithStatus` to reflect this change as well? Right now, `filterLocal` is only mentioned in the context of `startedBy`